charts/shoot-core/charts/csi-alicloud/templates/csi-attacher-dep.yaml

```diff
@@ -1,4 +1,68 @@
 {{- if .Values.enabled }}
+{{- if semverCompare ">= 1.13" .Values.kubernetesVersion -}}
+kind: Deployment
+apiVersion: {{ include "deploymentversion" . }}
+metadata:
+  name: csi-attacher
+  namespace: kube-system
+  labels:
+    origin: gardener
+    app: csi-attacher
+    garden.sapcloud.io/role: system-component
+    addonmanager.kubernetes.io/mode: Reconcile
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: csi-attacher
+  template:
+    metadata:
+      annotations:
+        # This, along with the CriticalAddonsOnly toleration below,
+        # marks the pod as a critical add-on, ensuring it gets
+        # priority scheduling and that its resources are reserved
+        # if it ever gets evicted.
+        scheduler.alpha.kubernetes.io/critical-pod: ''
```

Comment:
This annotation is deprecated as of `v1.13`.  `priorityClassName: system-cluster-critical` serves the same purpose.